.storybook/decorators/contextsWrapper.js

```diff
@@ -43,7 +43,15 @@ export const withContextWrapper = makeDecorator({
 			for (const s of scales) {
 				container.classList.toggle(`spectrum--${s}`, s === scale);
 			}
-		}, [color, scale, isExpress]);
+
+			if (args.staticColor === "black") {
+				container.style.backgroundColor = "rgb(181, 209, 211)";
+			} else if (args.staticColor === "white") {
+				 container.style.backgroundColor = "rgb(15, 121, 125)";
+			} else {
+				container.style.removeProperty("background-color");
+			}
```

Comment:
I'm not seeing any issues in the baseline or when I preview it locally. Can you share a brief test case of how to recreate what you're seeing?

Reply:
Here is the difference I was seeing. The "Default" in the Pending story should be displaying the same as the other two? Maybe "Default" also needs a set background color, or maybe it would be better to move the static pending to a Chromatic-only template for the static white story.
- [Turning on static color white control in the Pending story](https://64762974a45b8bc5ca1705a2-veqrwzrplp.chromatic.com/?path=/story/components-button--pending&args=staticColor:white)
<img width="308" alt="Screenshot 2024-03-26 at 10 26 23 AM" src="https://github.com/adobe/spectrum-css/assets/965114/a0da625c-8dca-4c01-a04d-ab2cc73ecbc7">

- [Static color white story, turning on Pending control](https://64762974a45b8bc5ca1705a2-veqrwzrplp.chromatic.com/?path=/story/components-button--static-color-white&args=isPending:!true)
<img width="301" alt="Screenshot 2024-03-26 at 10 20 41 AM" src="https://github.com/adobe/spectrum-css/assets/965114/633e8077-2c1a-49a2-8fc9-b6dcf29b7572">

Reply:
So I went into the button story and addressed the issue by aligning the test suite with the way I formatted actionbutton. It increases the scope of this PR a little bit but hopefully the alignment is helpful and ensures good coverage for both.

Reply:
Nice work adding that to button as well. I was just working on the stories for Button with the S2 migration and was wondering if what you added here was going to be needed.

I have one suggestion and one question about the Button story changes. 
1. Should we set the `variant` control arg to a default value, since the button requires one of them to be used? Without a variant class applied, it has some default colors that probably shouldn't be shown, and the `treatment` control will have no effect.
2. Are we okay with losing the Pending story that allowed the button to be clicked, to show the animation transition working? I believe @mdt2 also included this to help test that the contents of the button do not shift when pending is turned off and on.